distribution/conf/application.properties

```diff
@@ -97,10 +97,12 @@ db.pool.config.minimumIdle=2
 ### If turn on data loading task:
 # nacos.cmdb.loadDataAtStart=false
 
+#***********Metrics for tomcat **************************#
+server.tomcat.mbeanregistry.enabled=true
 
 #*************** Metrics Related Configurations ***************#
 ### Metrics for prometheus
-#management.endpoints.web.exposure.include=*
+management.endpoints.web.exposure.include=prometheus,health
```

Comment:
Can't default open.

Some orgnization or security engineer think the data will leak.